plugins/mission/mission.h

```diff
@@ -25,25 +38,97 @@ class Mission
         UNKNOWN
     };
 
+    /**
+     * Returns a human-readable English string for an Mission::Result.
+     */
     static const char *result_str(Result result);
 
+    /**
+     * Callback type for async mission calls.
+     */
     typedef std::function<void(Result)> result_callback_t;
 
-
+    /**
+     * Sends a vector of mission items to the device (asynchronous).
+     *
+     * The mission items are uploaded to a drone and can then be started and executed even if a
+     * connection is lost.
+     *
+     * @param mission_items reference to vector of mission items.
+     * @param callback callback to receive result of this request
+     */
     void send_mission_async(const std::vector<std::shared_ptr<MissionItem>> &mission_items,
                             result_callback_t callback);
 
+    /**
+     * Starts the mission (asynchronous).
+     *
+     * This needs the mission to be sent beforehand using send_mission_async.
+     *
+     * @param callback callback to receive result of this request
+     */
     void start_mission_async(result_callback_t callback);
+
+    /**
+     * Pauses the mission (asynchronous).
+     *
+     * When a mission is paused a drone should just hover at the spot and a fixedwing vehicle
```

Comment:
1. Where it says "drone" it should probably say "multicopter"
2. Might be worth saying that pausing a mission puts the vehicle into Hold mode, then linking to https://docs.px4.io/en/flight_modes/hold.html

Reply:
ok.